tests/Fixture/TagsFixture.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace SoftDelete\Test\Fixture;
+
+use Cake\ORM\Table;
+use Cake\TestSuite\Fixture\TestFixture;
+
+use SoftDelete\Model\Table\SoftDeleteTrait;
+
+class TagsTable extends Table
+{
+    use SoftDeleteTrait;
+
+    public $softDeleteField = 'deleted_date';
```

Comment:
I don't think making it private in the Table class is wise. User land code may well extend the table class. It also gives people the ability to change the field on the fly by updating the table property from other places in their application.